js/components/student-report-link.js

```diff
@@ -0,0 +1,30 @@
+import React, { PureComponent } from 'react'
+import studentReporturl from '../util/student-report-url'
+
+
+export default class StudentReportLink extends PureComponent {
+  constructor(props) {
+    super(props)
+  }
+
+  render() {
+    const {student, started} = this.props
+    const student_id = student.get('id')
+    const name = student.get('name')
+    const link = studentReporturl(student_id)
+    const linkToWork =
+      <a href={link} target="_blank">
+        Open {name}'s report
+      </a>
```

Comment:
Github diff is unhappy about this syntax, but I guess it works in the browser (right?). ;) Usually people put `( )` around the element or opening tag on the same line.